src/addremoveedit.js

```diff
@@ -52,10 +52,21 @@ export function saveone(element) {
   return todolist;
 }
 
-export function removeone(element) {
+// export function removeone(element) {
+//  console.log(element.parentNode.id)
+//   const todolist = load();
+//   const index = element.parentNode.id;
+//   todolist.splice(index, 1);
+//   save(todolist);
+//   return todolist;
+// }
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.